.env.example

```diff
@@ -13,3 +13,6 @@ BACKEND_PORT=3300
 NODE_ENV=development
 FRONTEND_HOST=http://localhost:3000
 FRONTEND_AUTHORIZED_URL=http://localhost:3000/authorized
+
+MAIL_SERVER_URL=https://mail.kir-dev.hu
```

Comment:
Remove this pls, we dont want to expose the service.

Reply:
😶‍🌫️